src/pages/ClerkScreen.tsx

```diff
@@ -96,11 +99,15 @@ const ClerkScreen = ({
                     <option value="" disabled>
                       Select a precinct for this device…
                     </option>
-                    {election.precincts.map(precinct => (
-                      <option key={precinct.id} value={precinct.id}>
-                        {precinct.name}
-                      </option>
-                    ))}
+                    {election.precincts
+                      .sort((a, b) =>
+                        alphaSort(a.name.toLowerCase(), b.name.toLowerCase())
```

Comment:
This is better done using [`String#localeCompare`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) like so:

```ts
.sort((a, b) =>
  a.name.localeCompare(b.name, undefined, {
    ignorePunctuation: true,
  })
)
```

Reply:
Great!